### DISCUSSION TITLE
Multi-user / Collaborative editing

### DISCUSSION THREAD

**Discussion by UnlimitedCookies** [eclipse-theia/theia#8486] [General] [answered]
Will Theia eventually support collaborative editing, similar to how it works in Google Docs for example?

**Comment by vince-fugnitto** [comment, marked as answer]
@UnlimitedCookies thank you for the discussion, we already have an issue tracking supporting collaboration (similarly to as you described), it may be best to continue discussion there instead:  https://github.com/eclipse-theia/theia/issues/2842.
